### PR TITLE
Changed cfncluster_version to parallelcluster_version

### PR DESCRIPTION
**Pull request by sean-smith**
```
$ grep -r "cfncluster_version" .
[nothing]
$ grep -r "cfncluster_cookbook_version" .
[nothing]
```

Signed-off-by: Sean Smith <seaam@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: amis/packer_alinux.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "variables" : {
-    "cfncluster_version" : "",
+    "parallelcluster_version" : "",
     "cfncluster_cookbook_version" : "",
```
Comment (lukeseawalker) [inline amis/packer_alinux.json]: are we keeping cfncluster here?

File: amis/packer_centos6.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "variables" : {
-    "cfncluster_version" : "",
+    "parallelcluster_version" : "",
     "cfncluster_cookbook_version" : "",
```
Comment (lukeseawalker) [inline amis/packer_centos6.json]: are we keeping cfncluster here?

File: amis/packer_centos7.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "variables" : {
-    "cfncluster_version" : "",
+    "parallelcluster_version" : "",
     "cfncluster_cookbook_version" : "",
```
Comment (lukeseawalker) [inline amis/packer_centos7.json]: are we keeping cfncluster here?

File: amis/packer_ubuntu1404.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "variables" : {
-    "cfncluster_version" : "",
+    "parallelcluster_version" : "",
     "cfncluster_cookbook_version" : "",
```
Comment (lukeseawalker) [inline amis/packer_ubuntu1404.json]: are we keeping cfncluster here?

File: amis/packer_ubuntu1604.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "variables" : {
-    "cfncluster_version" : "",
+    "parallelcluster_version" : "",
     "cfncluster_cookbook_version" : "",
```
Comment (lukeseawalker) [inline amis/packer_ubuntu1604.json]: are we keeping cfncluster here?

File: amis/packer_variables.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-  "cfncluster_version": "2.0.2",
+  "parallelcluster_version": "2.0.2",
   "cfncluster_cookbook_version": "2.0.2",
```
Comment (lukeseawalker) [inline amis/packer_variables.json]: are we keeping cfncluster here?

File: amis/packer_alinux.json (hunk 1 above)
Comment (sean-smith) [reply amis/packer_alinux.json]: removed

File: amis/packer_centos6.json (hunk 2 above)
Comment (sean-smith) [reply amis/packer_centos6.json]: removed

File: amis/packer_ubuntu1404.json (hunk 4 above)
Comment (sean-smith) [reply amis/packer_ubuntu1404.json]: removed

File: amis/packer_ubuntu1604.json (hunk 5 above)
Comment (sean-smith) [reply amis/packer_ubuntu1604.json]: removed

File: amis/packer_variables.json (hunk 6 above)
Comment (sean-smith) [reply amis/packer_variables.json]: removed

File: amis/packer_centos7.json (hunk 3 above)
Comment (sean-smith) [reply amis/packer_centos7.json]: removed

**Pull request merged**